MIGRATION.md

```diff
@@ -102,7 +103,7 @@
     - [Deploying build artifacts](#deploying-build-artifacts)
       - [Dropped support for file URLs](#dropped-support-for-file-urls)
       - [Serving with nginx](#serving-with-nginx)
-      - [Ignore story files from node_modules](#ignore-story-files-from-node_modules)
```

Comment:
Aha! So it's actually your editor who plays a back and forth with mine :D Some formatting rules are applied here, which doesn't seem to be necessary.